components/brave_rewards/resources/extension/brave_rewards/background/storage.ts

```diff
@@ -39,38 +29,6 @@ export const defaultState: RewardsExtension.State = {
   initializing: true
 }
 
-const cleanData = (state: RewardsExtension.State) => {
-  state = { ...state }
-  state.publishers = {}
-
-  const balance = state.balance as any
-  if (!balance || balance.total == null) {
-    state.balance = defaultState.balance
-  }
-
-  if (!state.parameters) {
-    state.parameters = defaultState.parameters
-  }
-
-  return state
-}
-
 export const load = (): RewardsExtension.State => {
-  const data = window.localStorage.getItem(keyName)
-  let state: RewardsExtension.State = defaultState
-  if (data) {
-    try {
-      state = JSON.parse(data)
-      state.initializing = true
-    } catch (e) {
-      console.error('Could not parse local storage data: ', e)
-    }
-  }
-  return cleanData(state)
+  return defaultState
```

Comment:
that is correct, because panel doesn't need store it and we will move towards non persistent state as extension can get offloaded and not be up to date